### PR TITLE
office 365 - fixing feed name and mocks

### PR DESCRIPTION
**Pull request by ronykoz**
## Status
- [] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

Fixing feed name as per Alexs' request 

@kirbles19 FYI

### CODE REVIEW

File: Packs/FeedOffice365/Integrations/FeedOffice365/FeedOffice365.py [hunk 1]
```diff
@@ -82,15 +82,15 @@ def build_iterator(self) -> List:
                 result.extend(indicators)
             except requests.exceptions.SSLError as err:
                 demisto.debug(str(err))
-                raise Exception(f'Connection error in the API call to Office365.\n'
+                raise Exception(f'Connection error in the API call to Office 365.\n'
```
Comment (bakatzir) [inline Packs/FeedOffice365/Integrations/FeedOffice365/FeedOffice365.py]:
```suggestion
                raise Exception(f'Connection error in the API call to {INTEGRATION_NAME}.\n'
```

File: Packs/FeedOffice365/Integrations/FeedOffice365/FeedOffice365.py [hunk 2]
```diff
@@ -82,15 +82,15 @@ def build_iterator(self) -> List:
...
                                 f'Check your not secure parameter.\n\n{err}')
             except requests.ConnectionError as err:
                 demisto.debug(str(err))
-                raise Exception(f'Connection error in the API call to Office365.\n'
+                raise Exception(f'Connection error in the API call to Office 365.\n'
```
Comment (bakatzir) [inline Packs/FeedOffice365/Integrations/FeedOffice365/FeedOffice365.py]:
```suggestion
                raise Exception(f'Connection error in the API call to {INTEGRATION_NAME}.\n'
```

File: Packs/FeedOffice365/Integrations/FeedOffice365/FeedOffice365.py [hunk 3]
```diff
@@ -82,15 +82,15 @@ def build_iterator(self) -> List:
...
                                 f'Check your Server URL parameter.\n\n{err}')
             except requests.exceptions.HTTPError as err:
                 demisto.debug(str(err))
-                raise Exception(f'Error issuing the request call to Office365.\n\n{err}')
+                raise Exception(f'Error issuing the request call to Office 365.\n\n{err}')
```
Comment (bakatzir) [inline Packs/FeedOffice365/Integrations/FeedOffice365/FeedOffice365.py]:
```suggestion
                raise Exception(f'Connection error in the API call to {INTEGRATION_NAME}.\n'
```

**Comment by bakatzir** [comment]
@yaakovi please see the insecure addition for the mocks
fixes: https://github.com/demisto/etc/issues/21181

**Comment by yaakovi** [comment]
forced merge, changing ID is not BC break as the integration is not out yet

**Pull request merged**